PhysicsTools/NanoAOD/python/autoNANO.py

```diff
@@ -28,6 +28,8 @@ def expandNanoMapping(seqList, mapping, key):
     # L1 flavours: add tables through customize, supposed to be combined with PHYS
     'L1' : {'customize': 'nanoL1TrigObjCustomize'},
     'L1FULL' : {'customize': 'nanoL1TrigObjCustomizeFull'},
+    # Muon POG flavours : add tables through customize, supposed to be combined with PHYS
+    'MUPOG' : {'customize' : 'PhysicsTools/NanoAOD/custom_muon_cff.PrepMuonCustomNanoAOD'},
```

Comment:
Hello, I think it would have been wiser for me to spot that `@PHYS` should be added by default here, if you are never going to ask `--step NANO:@MUPOG` alone (i.e not including the `@PHYS` part. we might need a quick PR for this

Reply:
@vlimant  we were using always --step @PHYS+@MUPOG even in the relval_nano.py. I can make a quick PR to change this.

Reply:
done in #44425

Reply:
can you please have a look at #44428 ?

Reply:
Hi @vlimant , I have posted a comment in #44428. The changes look fine to me. but please wait a little bit for other muon pog colleagues to confirm.